apps/compare_requests/views.py

```diff
@@ -76,20 +77,16 @@ def create(self, request, *args, **kwargs):
         pull_info = github.get_pull_request_info(repo_name, pull_req_num)
```

Comment:
This function has a total of 70 lines, that's quite lengthy and thus the code is not very readable. Wouldn't it make sense to split it into smaller abstractions? Your logger comments indicate the logical steps which I think would be a good starting point for division